R/messages.R

```diff
@@ -6,51 +6,47 @@ messages <- list(
 
     expectedTypeMsg <- paste0(expectedType, collapse = ", or ")
 
-    message <- paste0(
+    paste0(
       callingFunction, ": argument '", objectName,
       "' is of type '", type, "', but expected '", expectedTypeMsg, "'!", optionalMessage
     )
-
-    return(message)
   },
 
   errorGetEntityMultipleOutputs = function(path, container, optionalMessage = NULL) {
     # Name of the calling function
     callingFunctions <- sys.calls()
     callingFunction <- sys.call(-length(callingFunctions) + 1)[[1]]
 
-    message <- paste0(
+    paste0(

```

Comment:
No need to instantiate a local variable just to return it. In simple case, let's just use the standard R features here. There is no ambiguity